svthalia/concrexit
# Remove typo in GSuite-only help text

Closes #ISSUE



### Summary
Fix a small grammatical error

### How to test
Steps to test the changes you made:
1. Go to "Edit profile"
2. See that the help text for "Only receive Thalia emails on G Suite-account" is now correct
